content/AddedListPublicHealthInterventions.introduction.md

```diff
@@ -108,6 +108,8 @@ These can include characteristics such as when infected individuals are contagio
 The development of diagnostic tools is critical to this goal.
 Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
 Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
+Public health strategies to manage epidemics include anticipation and early detection of emerging diseases (aided by rapid development of diagnostics), containment (using strategies to test, trace and isolate such as widespread testing, contact tracing, quarantining and isolation), control and mitigation of spread (including social distancing), and elimination and eradication. 
+An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information), and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society [@url:https://www.who.int/emergencies/diseases/managing-epidemics-interactive.pdf].
```

Comment:
```suggestion
An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information), non-pharmaceutical measures to prevent and reduce transmission, and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society [@url:https://www.who.int/emergencies/diseases/managing-epidemics-interactive.pdf].
```